modules/common/src/main/mon.scala

```diff
@@ -246,11 +246,11 @@ object mon {
       val count = inc("puzzle.selector")
       val time = rec("puzzle.selector")
     }
-    object attempt {
-      val user = inc("puzzle.attempt.user")
-      val anon = inc("puzzle.attempt.anon")
-      val mate = inc("puzzle.attempt.mate")
-      val material = inc("puzzle.attempt.material")
+    object round {
+      val user = inc("puzzle.round.user")
+      val anon = inc("puzzle.round.anon")
+      val mate = inc("puzzle.round.mate")
+      val material = inc("puzzle.round.material")
```

Comment:
don't rename these strings, unless their meaning changes. Because it would lose history on the grafana monitoring. Keep `puzzle.attempt.XXX` whenever possible.